tcms/telemetry/templates/telemetry/testing/status-matrix.html

```diff
@@ -0,0 +1,100 @@
+                </label>
+                <div class="col-md-3">
+                    <select name="product" id="id_product" class="form-control selectpicker">
+                        <option value="">----------</option>
+                    </select>
+                </div>
+
+                <label for="id_select_version" class="col-md-1 col-lg-1">
+                    {% trans "Version" %}:
+                </label>
+                <div class="col-md-3">
+                    <select name="version" id="id_version" class="form-control selectpicker">
+                        <option value="">----------</option>
+                    </select>
+                </div>
+
+                <label for="id_select_build" class="col-md-1 col-lg-1">
+                    {% trans "Build" %}:
+                </label>
+                <div class="col-md-3">
+                    <select name="build" id="id_build" class="form-control selectpicker">
+                        <option value="">----------</option>
+                    </select>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label for="id_select_test_plan" class="col-md-1 col-lg-1">
+                    {% trans "Test Plan" %}:
+                </label>
+                <div class="col-md-3">
+                    <select name="test_plan" id="id_test_plan" class="form-control selectpicker">
+                        <option value="">----------</option>
+                    </select>
+                </div>
+
+                <label class="col-md-1 col-lg-1" for="id_after">{% trans "After" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <div class="input-group date-time-picker-pf">
+                        <input type="text" class="form-control" id="id_after">
+                        <span class="input-group-addon">
+                            <span class="fa fa-calendar"></span>
+                        </span>
+                    </div>
+
+                    {% include "datetimepicker_script.html" with selector="#id_after" %}
+                </div>
+
+                <label class="col-md-1 col-lg-1" for="id_before">{% trans "Before" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <div class="input-group date-time-picker-pf">
+                        <input type="text" class="form-control" id="id_before">
+                        <span class="input-group-addon">
+                            <span class="fa fa-calendar"></span>
+                        </span>
+                    </div>
+
+                    {% include "datetimepicker_script.html" with selector="#id_before" %}
+                </div>
+            </div>
+        </form>
+        <table class="table table-striped table-bordered table-with-horizontal-scroll">
+            <thead>
+            <tr>
+                <td class="tests-test-runs-header"> Tests/ Test runs</td>
```

Comment:
translation missing

Reply:
Change text to `Test case\Test run`, each of them marked for translation individually.

Reply:
Done. I will generate translation strings when we are ready to merge